sdk/docs/manually-written/sdk/reference/fixity.rst

```diff
@@ -0,0 +1,67 @@
+  operator has been declared as left-associative
+
+The *precedence* of operators defines, when combining different operators, which
+operator is processed first. For example, in general (and in Daml),
+multiplication *takes precedence* over addition. That is, ``x + y * z`` is
+parsed as ``x + (y * z)``. Operator precedence is expressed as a number, where a
+higher number indicates a higher precedence. Operators of same precedence are
+associated to the left (e.g. ``x + y - z`` is parsed as ``(x + y) - z``.
+
+The fixity and precedence of an operator are declared using the ``infixl``,
+``infix``, and ``infixr`` keywords (denoting left-, non-, and
+right-associativity, respectfully) that take an integer between 0 and 9
+inclusive and an operator the fixity applies to. For example, ``infixl 6 +``
+declares that ``+`` is a ``left-associative`` operator with precedence ``6``.
+These keywords can be used for user-defined operators as well. The following
+table shows the fixity and precedence for operators that are built-in to the
+Daml language, such as ``+`` and ``-``:
+
+==========  =========================================================  =======================================================================  ========================
+Precedence  Left-associative                                           Non-associative                                                          Right-associative
+==========  =========================================================  =======================================================================  ========================
+9           ```!``                                                                                                                              ``.``
+8                                                                                                                                               ``^``, ``^^``, ``**``
+7           ``*``, ``/``, ```div```, ```mod```, ```rem```, ```quot```
+6           ``+``, ``-``
+5                                                                                                                                               ``:``, ``++``
+4                                                                      ``==``, ``/=``, ``<``, ``<=``, ``>``, ``>=``, ```elem```, ```notElem```
+3                                                                                                                                               ``&&``
+2                                                                                                                                               ``||``
+1           ``>>``, ``>>=``
+0                                                                                                                                               ``$``, ``$!``, ```seq```
+==========  =========================================================  =======================================================================  ========================
```

Comment:
A quick grep gives:
```
infix 2 ==>
infix  4  ==, /=, <, <=, >=, >
infix 4 =/=
infix 4 ===

infixl 1 <&>
infixl 1 >>
infixl 1 >>=
infixl 4 *>
infixl 4 <*
infixl 4 <*>
infixl 4  <$
infixl 4 <$>
infixl 4 $>
infixl 6 +
infixl 6 -
infixl 7 %
infixl 7 *
infixl 7 /
infixl 9 !!

infixr 0 $
infixr 1 <=<, >=>
infixr 1 =<<
infixr 2  ||
infixr 2  |||
infixr 3  &&
infixr 3  &&&
infixr 5 
infixr 5 ++
infixr 5 ++
infixr 5 `pair`
infixr 5 `Pair`
infixr 6 <>
infixr 8 **
infixr 8 ^
infixr 8 <$$>
infixr 9 .
```
as the comprehensive list of daml-prim/daml-stdlib-provided infix operators, I think we can update this table for that.